src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/arm/activation.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/activation.hpp"
+#include "utils/cpu_test_utils.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace Activation {
+
+const auto basicCasesFp32 = ::testing::Combine(
```

Comment:
Looks like these cases are covered by common instances, isn't it?

Reply:
remove, thanks